### PR TITLE
Fix access list audit log formatting

### PR DESCRIPTION
**Pull request by kimlisa**
from :
image

fixed:
image

### CODE REVIEW

File: web/packages/teleport/src/services/audit/makeEvent.ts [hunk 1]
```diff
@@ -1575,3 +1587,14 @@ function truncateStr(str: string, len: number): string {
   }
   return str.substring(0, len - 3) + '...';
 }
+
+function formattedMembersTxt(members: { member_name: string }[]) {
```
Comment (gzdunek) [inline web/packages/teleport/src/services/audit/makeEvent.ts:1591]: A bit shorter option:
```suggestion
function formatMembers(members: { member_name: string }[]) {
```

File: web/packages/teleport/src/Audit/fixtures/index.ts [hunk 2]
```diff
@@ -3062,6 +3062,7 @@ export const events = [
         updated_by: 'mike',
```
Comment (gzdunek) [inline web/packages/teleport/src/Audit/fixtures/index.ts:3062]: Shouldn't we remove `updated_by` field from `members` in fixtures? `RawEventAccessList.members` doesn't contain it.

**Review by gzdunek — APPROVED** [review]

File: web/packages/teleport/src/services/audit/makeEvent.ts [hunk 3]
```diff
@@ -1575,3 +1587,14 @@ function truncateStr(str: string, len: number): string {
...
+  const memberNames = members.map(m => m.member_name);
+  const memberNamesJoined = memberNames.join(', ');
+
+  if (memberNames.length > 1) {
+    return `members [${memberNamesJoined}]`;
+  }
+
+  return `member [${memberNamesJoined}]`;
```
Comment (ravicious) [inline web/packages/teleport/src/services/audit/makeEvent.ts:1599]:
```typescript
import { pluralize } from 'shared/utils/text';
```

```suggestion
  return `${pluralize(memberNames.length, "member")} [${memberNamesJoined}]`
```

😎

File: web/packages/teleport/src/Audit/fixtures/index.ts
Comment (ravicious) [inline web/packages/teleport/src/Audit/fixtures/index.ts:1]: Could you add a fixture with multiple members?

**Review by ravicious — APPROVED** [review]

**Review by mdwn — APPROVED** [review]
Thanks for catching this!

**Pull request merged**